pandapower/run.py

```diff
@@ -241,6 +241,64 @@ def runpp(net, algorithm='nr', calculate_voltage_angles="auto", init="auto",
         _powerflow(net, **kwargs)
 
 
+def runpp_pgm(net, symmetric=True, algorithm="nr", error_tolerance_u_pu=1e-8, max_iterations=20, validate_input=False):
+    """
+        Runs powerflow using power-grid-model library
+
+        INPUT:
+            **net** - The pandapower format network
+
+        OPTIONAL:
+            **symmetric** (bool, True) -
+
+            - True: three-phase symmetric calculation, even for asymmetric loads/generations
+            - False: three-phase asymmetric calculation
+
+            **algorithm** (str, "nr") - Algorithms available in power-grid-model. Check power-grid-model documentation for detailed information on the algorithms.
+
+            - "nr" - Newton Raphson algorithm
+            - "bfsw" - Iterative current algorithm. Similar to backward-forward sweep algorithm
+            - "lc" - Linear current approximation algorithm
+            - "lin" - Linear approximation algorithm
+
+            **error_tolerance_u_pu** (float, 1e-8) - error tolerance for voltage in p.u.
+
+            **max_iterations** (int, 20) - Maximum number of iterations for algorithms. No effect on linear approximation algorithms.
+
+            **validate_input** (bool, False) - Validate input data to be used for power-flow conversion in power-grid-model. It is recommeneded to use pandapower.diagnostic tool prior.
+    """
+    try:
+        from power_grid_model_io.converters import PandaPowerConverter
+        from power_grid_model import PowerGridModel, CalculationType, CalculationMethod
+        from power_grid_model.validation import validate_input_data, errors_to_string
+        from power_grid_model.validation.errors import ValidationError
+    except ImportError:
+        raise ImportError(f"Import failed. Try using `pip install power-grid-model power-grid-model-io` to install the packages")
```

Comment:
Just installing `power-grid-model-io` would be enough as it will install `power-grid model` as a requirement. Installing both is more explicit, but `pip install power-grid-model-io` is shorter :) I don't have a strong preference, just wanted to point this out :)